binary_wheel_builder/api/build.py

```diff
@@ -114,22 +114,31 @@ def build_wheel(wheel_meta: Wheel, dist_folder: Path, worker_count: int = 1) ->
                 python_platform,
                 wheel_meta
             )
-            for python_platform in wheel_meta.platforms
-        ]
-        for future in concurrent.futures.as_completed(futures):
-            yield future.result()
-
+            for future in concurrent.futures.as_completed(futures):
+                if future.exception() is not None:
+                    print(f"Error in thread: {future.exception()}\n{traceback.format_exc()}")
+                else:
+                    try:
+                        yield future.result()
+                    except Exception as e:
+                        print(f"Error retrieving result from future: {e}\n{traceback.format_exc()}")
```

Comment:
chore: Rephrase this a bit more user-friendly, imho it is not relevant to the user that this was an error in a thread

Reply:
Yes, you´re right I have to think what another message to show for a better understanding, and maybe replace print with logging, thanks !